data/org.freedesktop.portal.DynamicLauncher.xml

```diff
@@ -153,7 +153,13 @@
           <varlistentry>
             <term>name s</term>
             <listitem><para>
-              The name chosen by the user for the launcher.
+              The name chosen by the user for the launcher or the predefined one.
```

Comment:
```suggestion
              The name chosen by the user for the launcher, or the predefined one.
```

Also: who chooses the "predefined" name? The portal implementation? The host?

Reply:
The caller of PrepareInstall can pass name/icon and they can edited by the user if the call to PrepareInstall allows them to

Reply:
So not sure if predefined is the correct wording